bin/study.js

```diff
@@ -1,5 +1,16 @@
 // Repeat Ruby Lab for JS
-// let myArray = [1, 2, 3, 4, 5];
+'use strict';
+let myArray = [1, 2, 3, 4, 5];
+
+myArray[myArray.length + 1] = 20;
+myArray[myArray.length - 3];
+myArray[9];
+myArray(myArray.length - 5, 3, [-12, -49]);
+myArray = [].concat.apply([], myArray);
```

Comment:
you are calling the method `concat` on a blank array. what would be the expected output?